icu4c/source/i18n/unicode/measunit.h

```diff
@@ -563,7 +563,7 @@ class U_I18N_API MeasureUnit: public UObject {
     /**
      * Returns by value, unit of acceleration: g-force.
      * Also see {@link #createGForce()}.
-     * @stable ICU 64
+     * @draft ICU 64
```

Comment:
Something is missing in the codebase, resulting in the "stable" annotations from dcfe6c4adae dropping back to "draft". Is it DRAFT_VERSIONS at the top of MeasureUnitTest.java that wasn't updated?